src/NuGet/Microsoft.Orleans.Development.nuspec

```diff
@@ -1,34 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
Yes I do. 

There are **only** two packages that need those files for each of the two "developer artifacts types" - one for grains and one for grain-interfaces. 

As you discovered yourself, it is easy to locate the tools .exe files from build .props . targets file IFF they are in the same NuGet package, but extremely hard to do that if they are spread across two NuGet packages. 

This latter point out-ways almost everything else in my mind, especially if you want to retain the flexibility of combining single packages with different versions. We already have a very strong tie between codegen.exe and the core code-gen interfaces in Orleans.dll, so i don't think we can realistically every undo that any time soon. 
For example, PR #194 created a breaking-change in the binary API between GrainReference abstract class and the type-specific generated grain-reference classes, which required matching changes on both Orleans.dll and codegen.exe.

I don't think we are planing to introduce any additional "developer artifact" types other than grin/grain-interfaces any time soon, and even if we do i suspect there will only even be a very small number of additions to deal with so i don't think the extra "de-normalization" really buys us anything here significant here.